composeApp/src/main/res/values/strings.xml

```diff
@@ -24,5 +24,24 @@
     <string name="meldingen_toestaan">Meldingen toestaan</string>
     <string name="meldingen_info">Charge Hub helpt je eraan denken wanneer het jouw beurt is om te laden</string>
     <string name="overslaan">Overslaan</string>
+    <string name="laadbeurten">Laadbeurten</string>
+    <string name="reservatie_info">Reserveer een laadbeurt door te kiezen uit snelle reservaties op de thuispagina of eentje aan te maken!</string>
+    <string name="laadbeurt_afstaan">Laadbeurt afstaan</string>
+    <string name="laadbeurt_afstaan_info">Elke laadbeurt kan afgestaan worden tot 15 min. voor hij van start gaat. Laattijdig afstaan zorgt voor een daling in level.</string>
+    <string name="level_systeem">Level systeem</string>
+    <string name="level_system_title">Je level wordt beïnvloedt door</string>
+    <string name="stiptheid">Stiptheid</string>
+    <string name="stiptheid_details">Op tijd laadbeurten beginnen en je voertuig verplaatsen.</string>
+    <string name="hoffelijkheid">Hoffelijkheid</string>
+    <string name="hoffelijkheid_details"><![CDATA[Ruilverzoeken accepteren & laadbeurten vroegtijdig of laattijdig afstaan.]]></string>
+    <string name="level_systeem_2_uitleg">Hoe hoger je huidige level, hoe meer functies je vrijspeelt zoals</string>
+    <string name="prioriteit">Prioriteit</string>
+    <string name="level_1">Level 1</string>
```

Comment:
Try and do this with a paramater so you don't have to create a string resource for every level